File: 1-codes/micro/micro04.c
#import<stdio.h>

int main(){
  int x, num, intervalo;

  for (x = 0; x < 5; x++) {
    printf("Digite um numero: ");
    scanf("%d", &num);
    if(num >= 10) {
      if (num <= 150) {
        intervalo = intervalo + 1;
      }
    }
  }

  printf("Ao total, foram digitados %d numeros no intervalotre 10 e 150.", intervalo);

	return 0;
}

File: 4-semantico/tests/micro04.py
def main() -> None:
    x = 0
    num = 0
    intervalo = 0
    for x in range(0,5):
        print("Digite o numero: ")
        int(input(num))
        if num >= 10:
            if num <= 150:
                intervalo = intervalo + 1
    print("\nAo total, foram digitados ")
    print(intervalo)
    print(" numeros no intervalotre 10 e 150")

main()
